paymentsheet/src/main/java/com/stripe/android/paymentelement/embedded/DefaultEmbeddedActivityLauncher.kt

```diff
@@ -0,0 +1,41 @@
+package com.stripe.android.paymentelement.embedded
+
+import androidx.activity.result.ActivityResultCaller
+import androidx.activity.result.ActivityResultLauncher
+import androidx.lifecycle.DefaultLifecycleObserver
+import androidx.lifecycle.LifecycleOwner
+import com.stripe.android.paymentsheet.model.PaymentSelection
+
+internal interface EmbeddedActivityLauncher {
+    fun launchForm(args: FormContract.Args, onSelected: (paymentSelection: PaymentSelection) -> Unit)
```

Comment:
I don't think you can pass `onSelected` here, this won't work for process death restoration.